pkgs/by-name/en/ente-cli/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildGoModule,
+  callPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+let
+  version = "0.2.1";
+in
+buildGoModule {
+  pname = "ente-cli";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "ente-io";
+    repo = "ente";
+    rev = "refs/tags/cli-v${version}";
+    hash = "sha256-gIDJUj2pn8rndXWN69bZdVfRLVB7AybXHMcioG2NI1k=";
+    fetchSubmodules = false;
+    sparseCheckout = [ "cli" ];
+  };
+
+  modRoot = "./cli";
+
+  vendorHash = "sha256-Gg1mifMVt6Ma8yQ/t0R5nf6NXbzLZBpuZrYsW48p0mw=";
+
+  CGO_ENABLED = 0;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.AppVersion=cli-v${version}"
+  ];
+
+  postInstall = ''
+    mv $out/bin/{cli,ente-cli}
```

Comment:
would be cool to add completion as well, the --help shows a `completion` command